index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">   
+        <link rel="stylesheet" href="styles.css">
+        <title>Embedding Images and Video</title>
+    </head>
+    <body>
+        <div class="header">
```

Comment:
I advise you to use HTML semantic elements for better [SEO](https://moz.com/beginners-guide-to-seo)